README.md

```diff
@@ -37,9 +37,10 @@ Liveblocks is a fully integrated solution built around core products, each
 enabling a different facet of collaborative experiences:
 [Presence](https://liveblocks.io/docs/products/presence),
 [Broadcast](https://liveblocks.io/docs/products/broadcast),
-[Storage](https://liveblocks.io/docs/products/storage), and Comments (private
-beta). You can decide what products you want to use based on your requirements
-and collaborative experiences you’re looking to build.
+[Storage](https://liveblocks.io/docs/products/storage), and
+[Comments (private beta)](https://liveblocks.io/docs/products/comments). You can
+decide what products you want to use based on your requirements and
+collaborative experiences you’re looking to build.
```

Comment:
```suggestion
[Storage](https://liveblocks.io/docs/products/storage), and
[Comments (private beta)](https://liveblocks.io/docs/products/comments). You can
decide which products you’d like to use based on the requirements of
the collaborative experiences you’re looking to build.
```
I try to avoid the word "want" in neutral-sounding documentation, it can sound _slightly_ negative as it's used for demanding something.